### PR TITLE
Fixed PHP 8.1 deprecations.

### PR DESCRIPTION
**Pull request by bartvanhoutte**

### CODE REVIEW

**Comment by pricop** [comment]
Is @S1lentium still around? Last activity on this repository was over 3 years ago. If not, perhaps we should run a separate fork that's maintained for the latest versions of PHP. I'm interested in this patch too.

**Comment by Forsakenrox** [comment]
im glad to see this fix, but i think maintainer is no more work on this library. It abandoned to years, so - the best options for us is make fork. Cant see anothersolutions =\

**Comment by simPod** [comment]
I have contacted the maintainer, he said he wishes to support the lib further.

**Review by simPod — APPROVED** [review]

File: src/Network.php [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <?php
 namespace IPTools;
 
+use ReturnTypeWillChange;
```
Comment (S1lentium) [inline src/Network.php:4]: What about this line? Does it cause a conflict with php 5?

File: src/Range.php [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <?php
 namespace IPTools;
 
+use ReturnTypeWillChange;
```
Comment (S1lentium) [inline src/Range.php:4]: What about this line? Does it cause a conflict with php 5?

**Review by S1lentium — CHANGES_REQUESTED** [review]

File: src/Network.php (hunk 1 above)
Comment (simPod) [reply src/Network.php:4]: No https://3v4l.org/BNoFr#v5.6.40

BTW we should drop php5 support as it's EOLed for a long time already

File: src/Range.php (hunk 2 above)
Comment (bartvanhoutte) [reply src/Range.php:4]: It does not.

**Pull request merged**